spec/fixtures/apple_verify_auth_code_response.json

```diff
@@ -0,0 +1 @@
+{"access_token":"aca3e04bbffa14ba1bbbdefe76c4afdb6.0.mrtwr.qKIemVfidy9dfzOBeQrazg","token_type":"Bearer","expires_in":3600,"refresh_token":"r10538d56ac494294936ebe0a3cea2b8b.0.mrtwr.TwQduM7CU-cCK5AGYj-Jdw","id_token":"eyJraWQiOiJZdXlYb1kiLCJhbGciOiJSUzI1NiJ9.eyJpc3MiOiJodHRwczovL2FwcGxlaWQuYXBwbGUuY29tIiwiYXVkIjoib3JnLmNydS5nb2R0b29scyIsImV4cCI6MTY4NjA2NzQyMywiaWF0IjoxNjg1OTgxMDIzLCJzdWIiOiIwMDEzNjEuYTVjYWZiN2Y0MmM4NDViODgwOWM0OGQwZjJiMDA4ODkuMTgwNCIsImF0X2hhc2giOiJmTjR1T1d3dG5GUXFkMmw5UEpKdklnIiwiZW1haWwiOiJsZXZpLmVnZ2VydEBnbWFpbC5jb20iLCJlbWFpbF92ZXJpZmllZCI6InRydWUiLCJhdXRoX3RpbWUiOjE2ODU5ODEwMDEsIm5vbmNlX3N1cHBvcnRlZCI6dHJ1ZX0.z-BCRqNqLO-jf_0ya8zczCLcA9Ukn2erV6-47RwWfr3gfZalIWXPV0RWH4e1CXYkAgYq6RrRv39Dyp62g_ddETV0jnvBLOrhzJ4-dQGJAWXwOonDM2BV0PMF1Tu_xiQnbcb12iZQ2Gom228krXowy4PcdCw4WRLGvRgSYESazuOrh3Ihvy3B7yifUb7Rs7gE4hpP4unonKfwxv7s_RzUasrAFYI4gvUjzfRryOGGBK1uR4hnQXl0Hc0oSkQv-01bknhYUtx3QsEFCF7J8v4P81eNsbl10qRa5zz3P2DOr8Fw8RHZtsduvUIZqn-WaCOWiBYzwXAQfLMY_5oaSZ6SIA"}
```

Comment:
just double checking that the refresh_token used in this sample is no longer actually valid with Apple? We don't want to accidentally leak a refresh_token that is effectively a credential for the mobile app

Reply:
Good call. I've invalidated it now:

```
{"error":"invalid_grant","error_description":"The token has expired or has been revoked."}
```